log/slow/parser.go

```diff
@@ -277,6 +284,9 @@ func (p *SlowLogParser) parseHeader(line string) {
 			} else if smv[1] == "Log_slow_rate_limit" {
 				val, _ := strconv.ParseUint(smv[2], 10, 64)
 				p.event.RateLimit = uint(val)
+			} else if smv[1] == "Thread_id" {
```

Comment:
The regexp tries to match `Id` or `Thread_id`. Shouldn't be the same here?

Reply:
Thanks for looking at this, I'll take a look today

Reply:
Good catch, I updated the regex to only match Id. This Thread_id is matched as part of the "metrics" parsing. It was probably innocuous but better to be safe.